frontend/index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8"/>
-    <meta content="width=device-width, initial-scale=1.0" name="viewport"/>
-    <title>Fairdrive Desktop App (BETA)</title>
-</head>
-<body>
-<div id="root"></div>
-<script src="./src/main.tsx" type="module"></script>
-</body>
+    <head>
+        <meta charset="UTF-8"/>
+        <meta content="width=device-width, initial-scale=1.0" name="viewport"/>
+        <title>Fairdrive Desktop App (BETA)</title>
```

Comment:
When did we add BETA to title (imho not needed) we have disclaimer  maybe we should add version to it, but even that is not to be included in App Title.